runtime/src/invoke_dpe.rs

```diff
@@ -45,18 +49,45 @@ impl InvokeDpeCmd {
             let command = Command::deserialize(&cmd.data[..cmd.data_size as usize])
                 .map_err(|_| CaliptraError::RUNTIME_INVOKE_DPE_FAILED)?;
             let flags = pdata.manifest1.header.flags;
+            let active_pl0_dpe_context_count = pdata.active_pl0_dpe_context_count;
+            let active_pl1_dpe_context_count = pdata.active_pl1_dpe_context_count;
```

Comment:
Now that we have APIs in DpeInstance, can we use those instead of storing this in persistent data.

Reply:
Done